x-pack/plugins/apm/typings/es_schemas/raw/fields/span_links.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface SpanLink {
+  trace: { id: string };
+  span: { id: string };
+}
+
+export type SpanLinks = SpanLink[];
```

Comment:
I'd prefer not to create a `SpanLinks` type. I think using array notation directly (`SpanLink[]`) is more readable. (e.g., I did not know whether this was an array, or a record, or something else when I saw the `SpanLinks` type).